src/views/Layout.jsx

```diff
@@ -4,9 +4,26 @@ import './Layout.css';
 import { auth } from '../api/config.js';
 import { useAuth, SignInButton, SignOutButton } from '../api/useAuth.jsx';
 import LoggedOut from '../components/LoggedOut.jsx';
+import { useEffect, useState } from 'react';
 
 export function Layout() {
 	const { user } = useAuth();
+	const [isMobile, setIsMobile] = useState();
+
+	useEffect(() => {
```

Comment:
Hey @bdanzer - I'd like to know if why we should avoid useEffect, if the only reason I'm wanting to use is simply for a function to be called at the end of page render.

It makes sense why you'd want to avoid it if it's tied to external systems, but in this use case, is it really worth the extra headache of avoiding it if you're only running a single function with such a small computational impact?

Reply:
I'm interested in Britt's response too. 
I was able to make the intended behavior happen with pure CSS.

Reply:
Came up with a css-less useEffect-less solution on the way home!